module/rest/src/config.ts

```diff
@@ -49,8 +49,11 @@ export class RestConfig {
 
   postConstruct() {
     if (!this.bindAddress) {
-      const is4 = !!os.networkInterfaces().lo.find(nic => nic.family === 'IPv4');
-      this.bindAddress = is4 ? '0.0.0.0' : '::';
+      let useIPv4 = false;
```

Comment:
In this case, `.find()` would be a better method to use since it will stop on first match.

```typescript
const useIPv4 = [...Object.values(os.networkInterfaces())].find(nic => 
  !!nic.find(iface => iface.family === 'IPv4'));
```

Reply:
Updated